web/controllers/package_controller.ex

```diff
@@ -3,24 +3,43 @@ defmodule HexWeb.PackageController do
 
   @packages_per_page 30
   @sort_params ~w(name downloads inserted_at)
+  @letters for letter <- ?A..?Z, do: <<letter>>
 
   def index(conn, params) do
+    letter        = HexWeb.Utils.safe_search(params["letter"])
     search        = HexWeb.Utils.safe_search(params["search"])
+    maintainer    = params["maintainer"]
+
+    filter =
+      cond do
+        letter ->
+          {:starts_with, letter}
+        maintainer ->
+          {:maintainer, maintainer}
```

Comment:
I don't like that people with the same name will be regarded as the same person. It can be confusing and actually a security risk if you think the Chris that publishes phoenix is the same person who publishes backdoor_package.

If we add searching it should be based on package owners and it's bigger discussion if we should make that information public. We need to update the privacy policy among other things.